Removed Bootstrap style link (version 3.3.7) and changed some elements styling

nicoleortizga opened the pull request:
I have removed Boostrap style link (version 3.3.7) and added the style link of the 4.0.0 version.
changed some elements styling (issue  #7 )

Commit:
Removed Bootstrap style link (version 3.3.7) and add style link of 4.0.0 version, changed some element styling

tulikavijay commented on templates/screen_test.html:
```diff
@@ -1,27 +1,25 @@
-}
-button{
-  margin-bottom:50px;
+.screen-form {
```
Can i see what it looks like. You can attach a screenshot

nicoleortizga replied:
![screen-form_1](https://user-images.githubusercontent.com/37906201/67488326-3d150c80-f635-11e9-86a0-ee59166af2c4.png)

![screen-form](https://user-images.githubusercontent.com/37906201/67488340-44d4b100-f635-11e9-8120-085f540fff56.png)

tulikavijay commented on templates/register.html:
```diff
@@ -26,15 +29,13 @@
-<div class="margins">
-<div class="heading">Thankyou for registering.</div>
-<div class="sub-heading"><a href={% url 'home' %}>Return to the home page</a></div>
+<h2>Thank you for registering.</h2>
```
I can't comprehend the size of the text, a screenshot would be great if possible for you.

nicoleortizga replied:
![register](https://user-images.githubusercontent.com/37906201/67488583-ab59cf00-f635-11e9-819a-20e5838e0b46.png)

tulikavijay commented on templates/navbar.html:
```diff
@@ -1,8 +1,9 @@
-<nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
-     <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+<nav class="navbar navbar-expand-lg navbar-inverse fixed-top navbar-dark bg-dark">
```
Can you explain the changes?

nicoleortizga replied:
Hi @tulikavijay, 
The responsive navbar toggle has been updated in bootstrap 4, we can specify the background color using bg- classes: `bg-light`, `bg-dark`, etc. and for _lighter_ text color we can use `navbar-dark`.

tulikavijay reviewed (commented):
Hey! i have added some comments for you. Please check it out.

tulikavijay commented:
Everything looks great! 
One thing i see though is the submit button on register page, it looks slightly disproportionate. I think that might be because of bootstrap4? Anyhow if you can fix it, it will be awesome. I'll merge your PR as soon as your done.

ghost commented:
Hi @tulikavijay 
How about this fix?
![register](https://user-images.githubusercontent.com/37906201/67580481-06f68c00-f70c-11e9-8117-19c604cd27a5.png)

Also, I saw that the login button is not styled. Can I make the change?
![login](https://user-images.githubusercontent.com/37906201/67580580-2b526880-f70c-11e9-8233-9a93924864eb.png)

tulikavijay commented:
This fix looks good 👍 
Also sure you can make changes to the login page.

Commit:
Added bootstrap 4 in settings.py and general UI changes

Added Bootstrap 4 in settings.py to render forms using Bootstrap 4 CSS classes.
